docs/docs/tutorial/chapter4/authentication.md

```diff
@@ -473,6 +473,11 @@ Yet another generator is here for you, this time one that will create pages for
 ```bash
 yarn rw g dbAuth
 ```
+:::warning
+
+In the dbAuth setup steps you will see the option for "Querying WebAuthn addition..." For now, either press enter to keep the default as 'false' or type 'no.' WebAuthn works, and is great, but we do not want to add it as part of the tutorial.  
```

Comment:
```suggestion
In the dbAuth setup steps you will see the option for "Querying WebAuthn addition..." For now, either press enter to keep the default as 'false' or type 'no'. WebAuthn works, and is great, but we do not want to add it as part of the tutorial.  
```

Reply:
@Tobbe FWIW, I usually add end of sentence punctuation inside of 'quotes' but outside of `codeblocks` that land at the end of sentences. I have thought about this in different ways, though. [It looks like with American english this is one way](https://www.thesaurus.com/e/grammar/does-punctuation-go-inside-or-outside-quotation-marks/), and with British english there are cases where this goes either way! I think I like this British rule better:

> In British English, periods and commas only go inside quotation marks if the original quoted material also had those punctuation marks.

Reply:
Ohh,  had no idea! That looks so weird to me 😆

In Swedish the rule is: If the sentence starts outside the quotation marks it also ends outside the quotation marks (so the period that ends the sentence goes outside the quotations). If the sentence starts within the quotation marks, it also ends inside the quotation marks (so the period, exclamation mark, etc goes inside the quotation marks).